src/ui/Windows/DboxMain.cpp

```diff
@@ -3115,14 +3118,24 @@ struct NoDuplicates{
 // Returns a list of entries as they appear in tree in DFS order
 void DboxMain::MakeOrderedItemList(OrderedItemList &OIL, HTREEITEM hItem)
 {
-  // Walk the Tree - either complete tree or only this group!
+  // Walk the Tree - either complete tree or only this group
```

Comment:
You can only export a group if you can select a group to export. Since groups are not displayed entities in the List view, the question is meaningless.